gridt/src/app/settings/change-email/change-email.page.ts

```diff
@@ -57,7 +58,7 @@ export class ChangeEmailPage{
       }
     );
 
-    timer(500).subscribe( () => this.SetService.getUserSettings());
+    timer(500).subscribe( () => this.SetService.updateUserSettings());
```

Comment:
In a different comment I state why I don't think that this timer is a good idea.